.github/workflows/ci.yml

```diff
@@ -588,6 +588,16 @@ jobs:
           echo "Tests: $TESTS"
           ./mvnw test ${{ env.SUREFIRE_RETRY }} -Pauth-server-quarkus -Pdb-${{ matrix.db }} "-Dwebdriver.chrome.driver=$CHROMEWEBDRIVER/chromedriver" -Dtest=$TESTS -pl testsuite/integration-arquillian/tests/base 2>&1 | misc/log/trimmer.sh
 
+      - name: Run cluster JDBC_PING2 smoke test
+        run: |
+          ./mvnw test ${{ env.SUREFIRE_RETRY }} \
+            -Pauth-server-cluster-quarkus \
+            -Pdb-${{ matrix.db }} \
+            -Dtest=RealmInvalidationClusterTest \
```

Comment:
I am re-using an existing cluster test here for simplicity, but I can add a dedicated smoke cluster test which just ensures that a cluster of size `x` is created as expected. The important thing is that we ensure a cluster is able to form with each of Keycloak's supported DBs.

More thorough testing of the JDBC_PING2 protocol and how it handles is cluster members leaving gracefully/abrubtly existing in the Infinispan repo [JdbcPing2IT](https://github.com/infinispan/infinispan/blob/main/server/tests/src/test/java/org/infinispan/server/persistence/JdbcPing2IT.java) [[JGroupsJdbcPing2IT.java](https://github.com/infinispan/infinispan/blob/main/server/tests/src/test/java/org/infinispan/server/persistence/JGroupsJdbcPing2IT.java)](https://github.com/infinispan/infinispan/blob/main/server/tests/src/test/java/org/infinispan/server/persistence/[JGroupsJdbcPing2IT.java](https://github.com/infinispan/infinispan/blob/main/server/tests/src/test/java/org/infinispan/server/persistence/JGroupsJdbcPing2IT.java))